ci/ci_check

```diff
@@ -191,27 +194,26 @@ if not (prcmd):
     else:
         print ('This will delete your previously cloned RTL repo plus all previously generated files')
         ask_user()
-        os.chdir('../cv32/sim/uvmt_cv32')
+        os.chdir(os.path.join(topdir, 'cv32/sim/uvmt_cv32'))
         os.system('make clean_all')
-        os.chdir('../core')
+        os.chdir(os.path.join(topdir, 'cv32/sim/core'))
         os.system('make clean_all')
-        os.chdir('../../../ci')
+        os.chdir(os.path.join(topdir, 'ci'))
 
 
 ################################################################################
 # script starts here
 
 # This script is run from the "ci" directory, but the paths used by simulator
 # commands assume we are at the root of the repo.
-os.chdir('../')
-topdir = os.getcwd()
+os.chdir(topdir)
 
 #
 # Step 1: Run the User Regression for the UVM verification environment.
 #
 # .metrics.json is the CI regression config used by Metrics CI tools.
 if (uvm):
-    with open('.metrics.json') as f:
+    with open(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '.metrics.json')) as f:
```

Comment:
This does not work for me. I get the following nasty-gram:
```
Traceback (most recent call last):
  File "./ci_check", line 216, in <module>
    with open(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '.metrics.json')) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/data/mike/GitHubRepos/openhwgroup/core-v-verif/intr_and_debug/../.metrics.json'
```

Reply:
I can replicate the failure.  Did not check the previous case where you actually run in the ci_check directory!